reports/gp2gp_placeholder_trending_report/gp2gp_placeholders_breakdown_graph_trending_report_count.splunk

```diff
@@ -0,0 +1,28 @@
+index="$index$"
+
+{% from 'gp2gp_placeholder_trending_report/gp2gp_placeholder_trending_report_base.splunk' import
+gp2gp_placeholder_report_trending_base %}
+
+{{ gp2gp_placeholder_report_trending_base() }}
+
+
+| WHERE numPlaceholders > 0
+| spath output=placeholders path=payload.ehr.placeholders{}
```

Comment:
this needs to be put above line 6 as it needs to be before the `transaction` function which is in the base query. In the past this has caused problems

Reply:
Not sure, but I will test this.